test/extreme.test.js

```diff
@@ -0,0 +1,131 @@
+'use strict'
+
+var test = require('tap').test
+var os = require('os')
+var writeStream = require('flush-write-stream')
+var spawn = require('child_process').spawn
+
+if (process.version.indexOf('v0.10') >= 0) {
+  require('tap').comment('skipped because of node v0.10')
+  process.exit(0)
+}
+
+test('extreme mode', function (t) {
+  var now = Date.now
+  var hostname = os.hostname
+  var proc = process
+  global.process = {
+    __proto__: process,
+    pid: 123456
+  }
+  Date.now = function () {
+    return 1459875739796
+  }
+  os.hostname = function () {
+    return 'abcdefghijklmnopqr'
+  }
+  delete require.cache[require.resolve('../')]
+  var pino = require('../')
+  var expected = ''
+  var actual = ''
+  var normal = pino(writeStream(function (s, enc, cb) {
+    expected += s
+    cb()
+  }))
+
+  var extreme = pino({extreme: true}, writeStream(function (s, enc, cb) {
+    actual += s
+    cb()
+  }))
+
+  var i = 44
+  while (i--) {
+    normal.info('h')
+    extreme.info('h')
+  }
+
+  var expected2 = expected.split('\n')[0]
+  var actual2 = ''
+
+  var e = 'global.process = { __proto__: process,  pid: 123456};Date.now = function () { return 1459875739796;};require("os").hostname = function () { return "abcdefghijklmnopqr";};var pino = require("./");var extreme = pino({extreme: true});extreme.info("h")'
```

Comment:
I know I actually added this in the first place, but since we're refactoring maybe we should clean this line up no the col width, i.e.

```
var e = 'global.process = {',
  '__proto__: process',
  'pid: 123456',
  '}',
  ....etc,
  ].join('')
```